github/Repository.py

```diff
@@ -2266,6 +2267,21 @@ def get_dir_contents(self, path, ref=github.GithubObject.NotSet):
         """
         return self.get_contents(path, ref=ref)
 
+    def get_codeowner_errors(self):
+        """
+        :calls: `GET /repos/{owner}/{repo}/topics <https://docs.github.com/en/rest/repos/repos#list-codeowners-errors>`_
+        :rtype: :class:`github.PaginatedList.PaginatedList` of :class:`github.CodeownerError.CodeownerError`
+        """
+        headers, data = self._requester.requestJsonAndCheck(
+            "GET",
+            f"{self.url}/codeowners/errors",
+        )
+        return [
+            github.CodeownerError.CodeownerError(self._requester, headers, element, completed=False)
+            for element in data["errors"]
+            if element is not None and "errors" in data.keys()
```

Comment:
Looks like the schema of `element` does not contain a `errors` property.
```suggestion
            if element is not None
```